.circleci/config.yml

```diff
@@ -1,16 +1,31 @@
-version: 2
+version: 2.1
+
+defaults: &defaults
+  docker:
+    - image: srclosson/grafana-plugin-ci-alpine:latest
+
 jobs:
   build:
-    docker:
-      - image: srclosson/grafana-plugin-ci-alpine:latest
+    <<: *defaults
     steps:
       - checkout
+      - restore_cache:
+          keys:
+            - go-modules-v1-{{ checksum "go.sum" }}
+            - go-modules-v1
+      - run:
```

Comment:
What's the benefit of downloading dependencies separately?

Reply:
Good point. Will fix